x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/transaction_detail/transaction_detail.spec.ts

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import url from 'url';
+import { synthtrace } from '../../../../synthtrace';
+import { opbeans } from '../../../fixtures/synthtrace/opbeans';
+
+const start = '2021-10-10T00:00:00.000Z';
+const end = '2021-10-10T00:15:00.000Z';
+
+const serviceOverviewPath = '/app/apm/services/opbeans-java/transactions/view';
+const baseUrl = url.format({
```

Comment:
nit: this isn't really the base url. It's the actual url for the page including params.

```suggestion
const pageUrl = url.format({
```